src/core/CMakeLists.txt

```diff
@@ -72,6 +73,7 @@ list(APPEND DCCORE_PUBLIC_HEADERS
   gestures/PinchGesture.h
   gestures/PinchGestureRecognizer.h
   ImageJpegDecompressor.h
+  ../log.h
```

Comment:
The include_directories(src) is still necessary due to inconsistencies of the folder layout in the source folder vs install folder. The dcWebservice inclusion in the core library is the point where it is still needed.